static/js/list_rendering.js

```diff
@@ -104,6 +104,7 @@ var list_render = (function () {
             // Needs to be enough to exceed the max height, so that a
             // scrollable area is created.
             init: function () {
+                ui.update_scrollbar($(".subscriber_list_container"));
```

Comment:
This `list_rendering.js` library is a generic library that is used for bunch of different modules throughout the codebase.  It definitely doesn't make sense for it to unconditionally update the subscriber list scrollbar whenever one of them is initialized :).

I think it would probably work to put this call just after `list_render()` is called to initialize one of these?  But if not, what you should do is add an extra "callback function" argument to the `list_render()`; and have `init()` or whatever part of the `list_render` infrastructure call that callback function when needed.  And then we'd have the subscriber management part of the system pass a little closure that does this `ui.update_scrollbar` call as the callback function into `list_render`.  Does that make sense?

@brockwhittaker FYI -- do you have thoughts on the design here?